src/pool/srv_pool.c

```diff
@@ -1112,6 +1322,19 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 		goto out;
 	event_cb_registered = true;
 
+	/* Update pool map by IV */
+	rc = pool_map_update(svc, map_version, map_buf);
+	if (rc) {
+		D_ERROR("pool_map_update failed "DF_RC"\n", DP_RC(rc));
+		D_GOTO(out, rc);
+	}
+
```

Comment:
Would you remove this blank line, please? I think the comment on line 1326 (after applying this PR) is supposed to describe to line 1336.

Reply:
ok